addons/element-remover/addon.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "Element remover",
+  "description": "Remove elements inside scratch",
+  "userstyles": [
+    {
+      "url": "username.css",
+      "matches": [
+        "https://scratch.mit.edu/projects/*"
+      ],
+      "if": {
+        "settings": { "username": true}
+      }
+    },
+    {
+      "url": "publish.css",
+      "matches": [
+        "https://scratch.mit.edu/projects/*"
+      ],
+      "if": {
+        "settings": { "publish": true}
+      }
+    }
+  ],
+  "tags": [
+    "community",
+    "projectPage",
+    "projectPlayer"
+  ],
+  "credits": [
+    {
+      "name": "Onako2",
+      "link": "https://scratch.mit.edu/users/Onako2"
+    }
+  ],
+  "latestUpdate": {
+    "version": "1.0.0"
+  },
```

Comment:
I don't think it's necessary to insert latestUpdate in the first iteration of the addon.